frontend/src/metabase/dashboard/actions/parameters.js

```diff
@@ -182,13 +173,27 @@ export const SET_PARAMETER_DEFAULT_VALUE =
 export const setParameterDefaultValue = createThunkAction(
   SET_PARAMETER_DEFAULT_VALUE,
   (parameterId, defaultValue) => (dispatch, getState) => {
-    updateParameter(dispatch, getState, parameterId, parameter =>
-      setParamDefaultValue(parameter, defaultValue),
-    );
```

Comment:
I removed setParamDefaultValue because I figured it wasn't doing much. You could have a different opinion that a parameter should be encapsulated though